css-canvas-medical/index.html

```diff
@@ -0,0 +1,143 @@
+                <p>
+                  Physician-owned primary care practices across the country have
+                  already made Canvas the fabric of their care.
+                </p>
+              </div>
+              <div class="row">
+                <div class="column-half">
+                  <p>Cascade Family Practice, Portland OR</p>
+                  <p>Temescal Crek Medicine, Oakland CA</p>
+                  <p>San Juan HealthCare, Friday Harbor WA</p>
+                </div>
+                <div class="column-half">
+                  <p>Ward Medical Clinic, Bonifay FL</p>
+                  <p>Caribou Medical, Towanda PA</p>
+                  <p>Bay Tree Family Care, Red Bay AL</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="full-column">
+            <div class="text">
+              <div class="learn">
+                <h2>Ready to learn more?</h2>
+                <p>Schedule a private meeting now</p>
+              </div>
+              <div class="get-started-section">
+                <button class="get-started">Get Started</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="full-column">
+            <div class="bottom-info">
+              <div class="bottom-info-container">
+                <div class="column-half">
+                  <div class="white-logo">
+                    <img
+                      src="images/logo-white.png"
+                      class="logo-white"
+                      alt="logo-white" />
+                  </div>
+                  <p>
+                    2307 Irving Street, Suite 228 San Franciso CA 94122 (800)
+                    370-1416 info@canvasmedical.com
+                  </p>
+                </div>
+                <div class="column-one-fourth">
+                  <div class="link-list">
+                    <a href="" class="link">Company</a>
+                    <a href="">About Us</a>
+                    <a href="">Blog</a>
+                  </div>
+                </div>
+                <div class="column-one-fourth">
+                  <div class="link-list">
+                    <a href="">Product</a>
+                    <a href="">Product</a>
+                    <a href="">Implementation</a>
+                    <a href="">For Health Plans</a>
+                    <a href="">Security</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="full-column">
+            <div class="bottom-links">
+              <div class="bottom-links-container">
+                <a href="">Privacy & Terms</a>
+                <a href="">Contact Us</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
It looks like you accidentally made each of these `row`s a child of the `row` element for the first `row` 🤔 . You should revise this and see if it has any effect on your output

Reply:
hi cody.  Thank you.  I just revised it.